### PR TITLE
fix: add user to group integration test

### PR DESCRIPTION
**Pull request by otlci**
Issue #, if available:

Description of changes:

Integration test fix.

Checklist:

* [x] Have you successfully deployed to an AWS account with your changes?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: workbench-core/example/app/src/routes/dynamicAuthorizationRoutes.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@
  */
 
 import { isPluginConfigurationError } from '@aws/workbench-core-authentication';
+import { isGroupNotFoundError } from '@aws/workbench-core-authorization';
```
Comment (bhudnell) [inline workbench-core/example/app/src/routes/dynamicAuthorizationRoutes.ts:7]: I think you'll get an eslint error about importing twice from the same source

File: workbench-core/example/app/src/routes/dynamicAuthorizationRoutes.ts [hunk 2]
```diff
@@ -18,7 +19,7 @@ import {
   CreateGroupRequestParser
 } from '@aws/workbench-core-authorization/lib/models/createGroup';
 import { validateAndParse } from '@aws/workbench-core-base';
-import { isRoleNotFoundError, isUserNotFoundError } from '@aws/workbench-core-user-management';
+import { isUserNotFoundError } from '@aws/workbench-core-user-management';
```
Comment (bhudnell) [inline workbench-core/example/app/src/routes/dynamicAuthorizationRoutes.ts:22]: this is reexported from the authorization package, use that one

File: workbench-core/example/app/src/routes/dynamicAuthorizationRoutes.ts [hunk 3]
```diff
@@ -65,15 +66,16 @@ export function setUpDynamicAuthorizationRoutes(router: Router, service: Dynamic
...
         }
         if (isUserNotFoundError(error)) {
-          throw Boom.tooManyRequests('User not found');
+          throw Boom.notFound('User not found');
         }
-        if (isRoleNotFoundError(error)) {
-          throw Boom.tooManyRequests('Role not found');
+        if (isPluginConfigurationError(error)) {
+          throw Boom.internal(error.message);
         }
```
Comment (bhudnell) [inline workbench-core/example/app/src/routes/dynamicAuthorizationRoutes.ts:77]: we dont need to check this here, `PluginConfigurationError`'s `IdpUnavailableError`'s and other internal errors are automatically turned into `Boom.internal` errors in the `unknownErrorHandler` function

Comment (otlci) [reply workbench-core/example/app/src/routes/dynamicAuthorizationRoutes.ts]: Removed.

**Review by bhudnell — APPROVED** [review]

**Pull request merged**